README.md

```diff
@@ -67,7 +69,7 @@ After writing this method, our new result should be:
 ...
 ```
 
-**think**: Where should we call `strip_punctuation`? Does it matter? Why?
+**think**: Where should we call `strip_punctuation`? Does it matter? Why? maybe we should have called it before passing the journal_entry to analyze_mood so it can match them with inlude? (otherwise the world to follow include has too many characters and they don't get counted)
```

Comment:
So I think I need to make this question more clear. What I'm trying to ask is "Should you call strip_punctuation inside the analyze_mood method or outside it?" You chose the latter. I would have chosen the former. Is there any case where you would want to analyze_mood without stripping the punctuation? This shows in that you had to repeatedly call strip_punctuation throughout your code (happy_days and overall_mood). It works the way you have it, but it's not super DRY.